src/wrapper/map.rs

```diff
@@ -35,3 +35,41 @@ pub fn map_put(env: NIF_ENV, map: NIF_TERM, key: NIF_TERM, value: NIF_TERM) -> O
     }
     Some(result)
 }
+
+pub struct MapIterator {
```

Comment:
Oh, it turns out I still had this little newtype in there. But that will be easy to rip out.